src/transforms/swimlanes.rs

```diff
@@ -71,7 +71,14 @@ inventory::submit! {
     TransformDescription::new::<SwimlanesConfig>("swimlanes")
 }
 
-impl GenerateConfig for SwimlanesConfig {}
+impl GenerateConfig for SwimlanesConfig {
+    fn generate_config() -> toml::Value {
+        toml::Value::try_from(Self {
+            lanes: IndexMap::new(),
+        })
+        .unwrap()
+    }
+}
```

Comment:
Should this transform also have a `generate_config` test like the others?

Reply:
Added!